cmake/BundledCivetWeb.cmake

```diff
@@ -30,8 +30,8 @@ function(use_bundled_civetweb SOURCE_DIR BINARY_DIR)
     endif()
 
     set(BYPRODUCTS
-            "lib/${PREFIX}civetweb.${SUFFIX}"
-            "lib/${PREFIX}civetweb-cpp.${SUFFIX}"
+            "${CMAKE_INSTALL_LIBDIR}/${PREFIX}civetweb.${SUFFIX}"
+            "${CMAKE_INSTALL_LIBDIR}/${PREFIX}civetweb-cpp.${SUFFIX}"
```

Comment:
I think you need to do the same on line 80, as well

Reply:
Good catch!!

Reply:
yeah -- that makes me wonder if adding `-DCMAKE_INSTALL_LIBDIR=lib` (as is done in BundledLibRdKafka) would be a better solution

but I think either way is fine